scout/constants/igv_tracks.py

```diff
@@ -1,32 +1,42 @@
 # igv.js track settings common for all users and all cases
+# ----- Human reference tracks -----
 HG19REF_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/hg19.fasta"
 HG19REF_INDEX_URL = (
     "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/hg19.fasta.fai"
 )
-HG19CYTOBAND_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/cytoBand.txt"
 HG38REF_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa"
 HG38REF_INDEX_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa.fai"
+
+# ----- Cytoband tracks -----
+HG19CYTOBAND_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg19/cytoBand.txt"
 HG38CYTOBAND_URL = (
     "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg38/cytoBandIdeo.txt"
 )
 
-HG38GENES_FORMAT = "gtf"
-HG38GENES_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg38/genes/Homo_sapiens.GRCh38.80.sorted.gtf.gz"
-HG38GENES_INDEX_URL = "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg38/genes/Homo_sapiens.GRCh38.80.sorted.gtf.gz.tbi"
+# ----- refSeq genes annotation tracks -----
 HG19GENES_FORMAT = "bed"
-HG19GENES_URL = (
-    "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg19/genes/refGene.hg19.bed.gz"
-)
-HG19GENES_INDEX_URL = (
-    "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg19/genes/refGene.hg19.bed.gz.tbi"
-)
+HG19GENES_URL = " https://s3-eu-west-1.amazonaws.com/pfigshare-u-files/30726131/NCBI_Refseq_genes_GRCh37_sorted.bed.gz"
```

Comment:
I've replace the links to the old RefSeq genes with my figshare links, same also below with the new GenCode tracks. The problem is that it says that access is denied:

![image](https://user-images.githubusercontent.com/28093618/132846069-36ac0ea4-0df2-471c-80e2-774828a2b2b4.png)

But it's weird, since the access to these tracks should be public, here: https://figshare.com/articles/dataset/NCBI_RefSeq_Human_Genes_tracks/16601285?file=30726191

If you are wondering how I can see that these are the right links to the files, try to use wget with the download links provided on the figshare pages